lib/appsignal/config.rb

```diff
@@ -147,6 +151,10 @@ def config_file
         root_path.nil? ? nil : File.join(root_path, 'config', 'appsignal.yml')
     end
 
+    def detect_from_system
+      self[:running_in_container] = true if Appsignal::System.container?
```

Comment:
That would throw the following warning when in a container:

```
/Users/tom/appsignal/agent/integrations/appsignal-ruby/lib/appsignal/config.rb:155: warning: string literal in condition
```

I hardly use the double bang anymore